### PR TITLE
Customer feature

### PR DESCRIPTION
**Pull request by GeorgeVragalev**

### CODE REVIEW

**Comment by GeorgeVragalev** [comment]
Added welcome message column
Admin can change welcome message for each user through admin panel
Welcome message is displayed in profile account
Default welcome message is set for a newly created user

File: src/Presentation/Nop.Web/Models/Customer/CustomerInfoModel.cs [hunk 1]
```diff
@@ -141,6 +141,10 @@ public CustomerInfoModel()
         public IList<AssociatedExternalAuthModel> AssociatedExternalAuthRecords { get; set; }
         public int NumberOfExternalAuthenticationProviders { get; set; }
         public bool AllowCustomersToRemoveAssociations { get; set; }
+        
+        //welcome message account info
```
Comment (denisburlacu) [inline src/Presentation/Nop.Web/Models/Customer/CustomerInfoModel.cs:145]: remove all unnecessary comment, if you want to add comment add via xml summary

File: src/Presentation/Nop.Web/Models/Customer/RegisterModel.cs [hunk 2]
```diff
@@ -141,6 +141,10 @@ public RegisterModel()
         public string VatNumber { get; set; }
         public bool DisplayVatNumber { get; set; }
 
+        //welcome message account info
+        [NopResourceDisplayName("Account.Fields.WelcomeMessage")]
```
Comment (denisburlacu) [inline src/Presentation/Nop.Web/Models/Customer/RegisterModel.cs:145]: same here as for CustomerInfoModel

File: src/Presentation/Nop.Web/Areas/Admin/Controllers/CustomerController.cs [hunk 3]
```diff
@@ -568,7 +571,10 @@ public virtual async Task<IActionResult> Edit(CustomerModel model, bool continue
                 {
                     customer.AdminComment = model.AdminComment;
                     customer.IsTaxExempt = model.IsTaxExempt;
-
+                    
+                    //welcome message
```
Comment (denisburlacu) [inline src/Presentation/Nop.Web/Areas/Admin/Controllers/CustomerController.cs]: remove unnecessary comment, when i read line 576 for me is clear that this is welcome message

Comment (denisburlacu) [reply src/Presentation/Nop.Web/Areas/Admin/Controllers/CustomerController.cs]: and check all other places

File: src/Presentation/Nop.Web/Areas/Admin/Models/Customers/CustomerModel.cs [hunk 4]
```diff
@@ -148,6 +148,9 @@ public CustomerModel()
 
         [NopResourceDisplayName("Admin.Customers.Customers.Fields.AdminComment")]
         public string AdminComment { get; set; }
+        
+        [NopResourceDisplayName("Admin.Customers.Customers.Fields.WelcomeMessage")]
```
Comment (denisburlacu) [inline src/Presentation/Nop.Web/Areas/Admin/Models/Customers/CustomerModel.cs:152]: add localisation for this in xml file

**Review by denisburlacu — APPROVED** [review]

**Pull request merged**